onnxruntime/contrib_ops/cuda/bert/attention_impl.cu

```diff
@@ -540,7 +540,9 @@ Status QkvToContext(
   float zero = 0.f;
 
   // For raw attention mask, the scalar 1/sqrt(H) is moved to combine with softmax computation.
-  const float rsqrt_head_size = 1.f / sqrt(static_cast<float>(qk_head_size));
+  const float norm_factor = parameters.norm_factor;
+  const float rsqrt_head_size = norm_factor == 0.0f ? 1.f / sqrt(static_cast<float>(qk_head_size))
```

Comment:
norm_factor shall not be used to calculate rsqrt_head_size (just look at the name and we know what it is).

Reply:
fixed